### DISCUSSION TITLE
Can we add custom fields?

### DISCUSSION THREAD

**Discussion by matwalk** [amundsen-io/amundsen#1680] [Q&A]
Hi everyone,
I'm new to amundsen. I would like to know how far can we customize the metadata model.
For example, if I want to add a new field to describe a table (ex:  a classification for personnal data,  a status etc...)

Is it easy to add it? Do you have a documentation about this?

Thank you for your help

**Comment by VoLKyyyOG** [comment]
Yep, theres no docs but I'm merging a new PR that covers it. See:
- https://github.com/amundsen-io/amundsen/pull/1903
- https://github.com/amundsen-io/amundsen/discussions/1828

for details.
